### PR TITLE
Feature/nex 107 add tao core sdk dependencies

### PR DESCRIPTION
**Pull request by btamas**
Initial version with dependencies of `tao-core-sdk`.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "core"
+  ],
+  "license": "GPL-2.0",
+  "bugs": {
+    "url": "https://github.com/oat-sa/tao-core-libs-fe/issues"
+  },
+  "homepage": "https://github.com/oat-sa/tao-core-libs-fe#readme",
+  "dependencies": {
+    "handlebars": "^1.3.0",
+    "jquery": "^1.12.4",
```
Comment (krampstudio) [inline package.json]: the jquery is version is 1.9.1

Comment (krampstudio) [reply package.json]: I prefer to perform the update as a dedicated task

File: package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    "library",
+    "core"
+  ],
+  "license": "GPL-2.0",
+  "bugs": {
+    "url": "https://github.com/oat-sa/tao-core-libs-fe/issues"
+  },
+  "homepage": "https://github.com/oat-sa/tao-core-libs-fe#readme",
+  "dependencies": {
+    "handlebars": "^1.3.0",
```
Comment (krampstudio) [inline package.json]: I think we should use the exact versions and control them

File: README.md [hunk 3]
```diff
@@ -1,2 +1,6 @@
-# tao-core-libs-fe
+# tao-core-libs
 TAO Frontent Core Libraries
+
+Available scripts in the project:
+
+- `npm run build`: put libraries in AMD format into `dist` directory
```
Comment (krampstudio) [inline README.md]: Could you please add a section : `How to add a new library` so people doesn't forget to expose it if coming from `node_modules`

**Comment by krampstudio** [comment]
It's a good start, but figure out that using the paths from `src` isn't really convenient. I'd prefer to have 
```js
import { jquery, lodash } from '@oat-sa/tao-core-libs';
```
instead of
```js
import jquery from '@oat-sa/tao-core-libs/src/jquery';
import lodash from '@oat-sa/tao-core-libs/src/lodash';
```
and to maintain the modules export into an `index.js` like
```js
module.exports = {
   jquery : require('jquery'),
  lodash : require('lodahs'),
}
```
knowing that modern bundlers with "tree shaking"  will be able to bundle only the actually used library.

**Comment by btamas** [comment]
> It's a good start, but figure out that using the paths from `src` isn't really convenient. I'd prefer to have
> 
> ```js
> import { jquery, lodash } from '@oat-sa/tao-core-libs';
> ```
> 
> instead of
> 
> ```js
> import jquery from '@oat-sa/tao-core-libs/src/jquery';
> import lodash from '@oat-sa/tao-core-libs/src/lodash';
> ```
> 
> and to maintain the modules export into an `index.js` like
> 
> ```js
> module.exports = {
>    jquery : require('jquery'),
>   lodash : require('lodahs'),
> }
> ```
> 
> knowing that modern bundlers with "tree shaking" will be able to bundle only the actually used library.

I have already tried this solution, but it's not working if I integrate this module into `tao-core-ui` and build it to AMD format. I'm open for any idea how can we solve this.

**Comment by krampstudio** [comment]
> > It's a good start, but figure out that using the paths from `src` isn't really convenient. I'd prefer to have
> > ```js
> > import { jquery, lodash } from '@oat-sa/tao-core-libs';
> > ```
> > 
> > 
> > instead of
> > ```js
> > import jquery from '@oat-sa/tao-core-libs/src/jquery';
> > import lodash from '@oat-sa/tao-core-libs/src/lodash';
> > ```
> > 
> > 
> > and to maintain the modules export into an `index.js` like
> > ```js
> > module.exports = {
> >    jquery : require('jquery'),
> >   lodash : require('lodahs'),
> > }
> > ```
> > 
> > 
> > knowing that modern bundlers with "tree shaking" will be able to bundle only the actually used library.
> 
> I have already tried this solution, but it's not working if I integrate this module into `tao-core-ui` and build it to AMD format. I'm open for any idea how can we solve this.

**Comment by krampstudio** [comment]
sorry wrong buttons

**Comment by krampstudio** [comment]
I'm closing this one and will reopen a new one for better follow up

**Pull request closed without merging**